src/useCreateComponent.tsx

```diff
@@ -1,12 +1,17 @@
 /* eslint-disable react-hooks/exhaustive-deps */
 import * as React from 'react';
 import {useConnectComponents} from './ConnectComponents';
-import {ConnectElementTagName} from '@stripe/connect-js';
+import {
+  ConnectElementTagName,
+  ConnectHTMLElementRecord,
+} from '@stripe/connect-js';
 
-export const useCreateComponent = (
-  tagName: ConnectElementTagName
-): {wrapper: JSX.Element; component: HTMLElement | null} => {
-  const [component, setComponent] = React.useState<HTMLElement | null>(null);
+export const useCreateComponent = <T extends ConnectElementTagName>(
```

Comment:
@kaiying-stripe  Did you mean to update the connect.js version in the package.json?

Reply:
Yeah, I forgot to push that 😅